pkg/render/monitor/monitor_test.go

```diff
@@ -130,6 +130,70 @@ var _ = Describe("monitor rendering tests", func() {
 		Expect(service.Labels["k8s-app"]).To(Equal("tigera-prometheus"))
 	})
 
+	It("Should render Prometheus resources with resources requests and limits", func() {
```

Comment:
Would be good to include the init container.

Reply:
We don't inject any initcontainer for prometheus or alertManger.